static/css/dashboard/settings.css

```
@@ -0,0 +1,680 @@
+
+.btn:disabled {
+    opacity: 0.5;
+    cursor: not-allowed;
+    pointer-events: none;
+}
+
+.btn-settings {
+    background: var(--accent-primary);
+    color: white;
+    padding: 4px 8px;
+    font-size: 12px;
```

Stylelint (17.6.0)

[error] Unexpected duplicate "font-size" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)